asciidoctorj-core/src/main/java/org/asciidoctor/jruby/internal/AsciidoctorUtils.java

```diff
@@ -86,7 +86,8 @@ private static List<String> getOptions(Map<String, Object> options) {
             optionsAndAttributes.add(options.get(Options.ERUBY).toString());
         }
 
-        if (options.containsKey(Options.HEADER_FOOTER)) {
```

Comment:
I wonder if we can remove this entire method.
It seems like it's only called to produce some debug log message and in one test. 
But apart from that AsciidoctorUtils.toAsciidoctorCommand() and AsciidoctorUtils.getOptions() does not seem to provide much value.

Reply:
If you mean `toAsciidoctorCommand` I think the same. Whenever I wanted to reproduce something in asciidoctor I could build the command directly, no need to check this debug. And it's a source of modularity issues.

Do you want to create an issue for that? It helps me to keep track of changes in multiple branches.